### ISSUE TITLE
Supported Platform of RHEL is not suffcient

### ISSUE THREAD

**Issue by ronking-zhu** [MicrosoftDocs/sql-docs#7153]
[Enter feedback here]
In the Supported Platform section, Red Hat Enterprise Linux 8.4-8.5 should supported SQL Server also after confirmed with Product group Tejas. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3f0bcec3-7063-e556-e089-f6df587e2f03
* Version Independent ID: 801bc1d0-3453-9fba-2d82-d913ca90fdc8
* Content: [Installation guidance for SQL Server on Linux - SQL Server](https://docs.microsoft.com/en-us/sql/linux/sql-server-linux-setup?view=sql-server-ver15)
* Content Source: [docs/linux/sql-server-linux-setup.md](https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/linux/sql-server-linux-setup.md)
* Product: **sql**
* Technology: **linux**
* GitHub Login: @VanMSFT
* Microsoft Alias: **vanto**

**Comment by WilliamAntonRohm** [comment]
@ronking-zhu -- thank you for your feedback.

@VanMSFT -- please look into this issue.

**Comment by VanMSFT** [comment]
@tejasaks - Hey Tejas, can you please confirm?

**Comment by tejasaks** [comment]
Adding @amvin87 as well. 

We will confirm and get back on this.

**Comment by tejasaks** [comment]
@VanMSFT , we have confirmed that we have validated everything till RHEL 8.4. The RHEL 8.5 is a minor release and will be supported for short timeframe only so we will validate everything again on RHEL 8.6 which has a longer support lifespan. Though RHEL 8.5 being a minor release with fewer changes, it should be backward compatible and work well with SQL Server. @amvin87 for any additional thoughts here.
